gufe/mapping/componentmapping.py

```diff
@@ -21,3 +21,13 @@ def __init__(self, componentA: gufe.Component, componentB: gufe.Component):
 
     def __contains__(self, item: gufe.Component):
         return item == self._componentA or item == self._componentB
+
+    @property
+    def componentA(self) -> gufe.Component:
+        """A copy of the first Component in the mapping"""
```

Comment:
It's not really a copy though right? It's just giving you the object itself.

Reply:
True! fixed.